examples/web-color/README.md

```diff
@@ -0,0 +1,136 @@
+# Web color example
+
+This example shows the mechanics of applying patches to a register. The data
+is the list of web colors as defined by the [W3C](https://www.w3c.org), in
+particular [HTML4](https://www.w3.org/TR/1999/REC-html401-19991224/) and
+[CSS Color Module Level 3](https://www.w3.org/TR/css-color-3/).
+
+
+## Setup
+
+Ensure that you have Python +3.6 installed on your machine.
+
+Then, install the `registers` command line interface:
+
+```sh
+pip3 install --user "git+https://github.com/openregister/registers-cli#egg=registers"
+```
+
+Then verify it works:
+
+```sh
+$ registers --help
+Usage: registers [OPTIONS] COMMAND [ARGS]...
+
+  registers lets you manage a Register represented as a Register
+  Serialisation Format (RSF).
+```
+
+## Structure of the example
+
+This example has:
+
+* The **register** for the web colors in `src/web-color.rsf`.
+* The **data** in TSV we will add to the register in `data/`.
+
+Note that the filenames could be anything, the content is what's relevant.
+
+
+## The initial metadata
+
+A register needs some metadata before you can add data into it. In particular:
+the register identifier and the schema that any data needs to conform to.
+
+Let's see the metadata available in the web color register:
+
+```sh
+registers context show src/web-color.rsf
+```
+
+And the schema:
+
+```sh
+registers schema show src/web-color.rsf
+```
+
+From this we know that any data we want to add must have the primary key
+`web-color` (which happens to be the registers identifier as well) and can
+have any of: `name`, `start-date`, `end-date`.
+
+
+We can also check that in fact there is no data yet:
+
+```sh
+registers record list src/web-color.rsf
+```
+
+
+## The first patch
+
+The first tsv, `data/01.tsv`, has the list of web colors defined in HTML4.
+
+To be able to apply the patch to the register we need to transform the TSV
+into an RSF patch:
+
+```sh
+registers patch create --rsf src/web-color.rsf data/01.tsv > patch_01.rsf
+```
+
+Now you can apply the patch to the register:
+
+```sh
+registers patch apply --rsf src/web-color.rsf patch_01.rsf
+```
+
+This last command _appends_ the content of the patch to the register so now
+we have some records to list. Let's do it in CSV:
```

Comment:
I found this wording confusing. Maybe something like _So now let's preview the records as CSV_ would be clearer?

Reply:
Nice one, I'll change it